susemanager-utils/susemanager-sls/salt/clusters/addnode.sls

```diff
@@ -0,0 +1,24 @@
+{%- if pillar.get('ssh_auth_sock', False) %}
+ssh_agent_socket:
```

Comment:
Please start all our ids with `mgr_` and use also a unique name.
If somebody call `addnode` and `removenode` in one `state.apply` it will fail.
I think it is better to use always unique ids.